Editorial review: Add docs for scroll-state container queries (#38216)
mdn/content: files/en-us/glossary/scroll_snap/index.md

  - : The **snapport** ​is the area of the [scrollport](/en-US/docs/Glossary/Scroll_container#scrollport) that is used as the alignment container for the scroll snap areas when calculating snap positions. By default, it is the same as the scroll container's visual viewport, but is the area of the scrollport defined by the {{CSSXref("scroll-padding")}} property. The [snap areas](#snap_area) are aligned to the scrollport.

- `Snap area`

  - : The **snap area** of an element in a scroll container is the area defined using {{CSSXref("scroll-margin")}} outsets specified on the element. The snap area is used to snap the element to its snapport.

- `Snap target`

>   - : A descendant element inside a scroll snap container that is snapped to when the container scrolls. You can define the exact position that is snapped to on each snap target by setting a {{CSSXref("scroll-snap-align")}} property value on it.

Reviewer: ```suggestion
  - : A descendant element inside a scroll snap container that is snapped to when the container scrolls. The {{CSSXref("scroll-snap-align")}} property defines the snap position of each snap target.
```
glossary lingo rather than tutorial or guide.
